Cargo.toml

```diff
@@ -169,7 +169,7 @@ indicatif = "0.16"
 insta = { version = "1.21.0", features = ["toml"] }
 is-terminal = "0.4"
 itertools = "0.12"
-jsonwebtoken = { version = "8.1.0" }
+jsonwebtoken = { git = "https://github.com/jsdt/jsonwebtoken.git", rev = "7f0cef63c74f58dfa912e88da844932ba4c71562"}
```

Comment:
NIT: We should move this fork under Clockwork Labs in another PR.

Reply:
Either that or switch to a better jwt library.

Reply:
That too is an option.